src/gc/unix/cgroup.cpp

```diff
@@ -119,10 +119,11 @@ class CGroup
             return true;
         }
         
-        cpu_count = quota / period;
+        cpu_count = (double) quota / period;
         if (cpu_count < UINT32_MAX)
         {
-            *val = cpu_count;
+            // round up
+            *val = (uint32_t)(cpu_count + 0.999999999);
```

Comment:
@luhenry I think in practice we won't ever get near `UINT32_MAX`, but this rounding up should be above the `< UINT32_MAX`  check.

Reply:
Fixed with checking against `UINT32_MAX - 1`